tests/specs/session-replay/initialization.e2e.js

```diff
@@ -0,0 +1,65 @@
+import { config, getSR } from './helpers'
+import { testRumRequest } from '../../../tools/testing-server/utils/expect-tests'
+
+export default (function () {
+  describe('Session Replay Initialization', () => {
+    afterEach(async () => {
+      await browser.destroyAgentSession(browser.testHandle)
+    })
+
+    describe('Feature flags', () => {
+      // this test needs to be able to override the SR flag to 0
+      // work needed to enable this in WDIO
```

Comment:
Is this comment still valid? Can you elaborate on what we are missing in WDIO to make this possible?

Reply:
this comment needs to be removed, thanks